.gitignore

```diff
@@ -1,9 +1,4 @@
-d_clear
-d_do
-testing_file
-proj.xcodeproj
+package.json
```

Comment:
This file shouldn't be in this project 🤔

Reply:
Why not? I think of `yarn` in a similar way I think of `make`. It's a simple way to write cross-shell scripts.

Reply:
But docker version does not use any Node package... so what would be inside of this file? And if it's ignored it won't be delivered to the end users

Reply:
I use it to start the docker version. Clean the docker version. Rebuild the docker version. Yarn is not only for use with node packages. It's a general purpose, cross-shell scripting engine.